frontend/packages/core/src/accordion.tsx

```diff
@@ -17,6 +17,8 @@ const StyledAccordion = styled(MuiAccordion)({
   boxShadow: "none",
   border: "1px solid transparent",
   boxSizing: "content-box",
+  maxWidth: "100%",
+  overflowWrap: "anywhere",
```

Comment:
breaks the string at an arbitrary point or where there is a natural break. otherwise, the cells in the row won't share the space evenly. example

<img width="561" alt="Screen Shot 2021-01-07 at 5 13 42 PM" src="https://user-images.githubusercontent.com/39421794/103950947-ccf48880-510b-11eb-924e-55772a9662cd.png">